adlfs/spec.py

```diff
@@ -287,6 +293,7 @@ def __init__(
             ]
         self.location_mode = location_mode
         self.credential = credential
+        if account_host: self.account_host = account_host
```

Comment:
This will probably fail our linting on the formatting. You can run either `pre-commit run --all-files` or `black` locally to fix that.

Reply:
Done.